apps/electron-app/src/main/menu/index.ts

```
@@ -1,59 +1,467 @@
+      {
+        label: "Force Close All Dialogs",
+        accelerator: isMac ? "Command+Shift+Escape" : "Control+Shift+Escape",
+        click: () => {
+          // Force close all dialogs by accessing the dialog manager
+          const focusedWindow = BrowserWindow.getFocusedWindow();
+          if (focusedWindow) {
+            const appWindow = browser.getApplicationWindow(
+              focusedWindow.webContents.id,
+            );
+            if (appWindow && appWindow.dialogManager) {
+              appWindow.dialogManager.closeAllDialogs();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)